examples/fastify-ssr/package.json

```diff
@@ -20,7 +20,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@reach/router": "^1.2.1",
```

Comment:
`npm i` was failing because @reach/router ([even in its latest version](https://github.com/reach/router/blob/master/package.json#L29)) has a peer dependency of React < 17